### PR TITLE
News entries for ion-js 2.0 and ion-hash-js 4.0

### PR DESCRIPTION
**Pull request by pbcornell**
Also updates the Libraries page and addresses minor formatting for recent ion-java news item.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: _posts/2020-02-26-ion-java-1_6_0-released.md [hunk 1]
```diff
@@ -9,6 +9,5 @@ This release includes:
 * Optimizations for reading and writing binary strings.
 * A fix for a bug that could cause an infinite loop when an IonValue was modified while simultaneously being read with a TreeReader.
 
-| [Release Notes](https://github.com/amzn/ion-java/releases/tag/v1.6.0)
-| [Ion Java](https://github.com/amzn/ion-java) |
+| [Release Notes](https://github.com/amzn/ion-java/releases/tag/v1.6.0) | [Ion Java](https://github.com/amzn/ion-java) |
```
Comment (zslayton) [inline _posts/2020-02-26-ion-java-1_6_0-released.md:12]: Just an observation: this doesn't change the way the file is rendered, but it does make this line of source long enough that it has to wrap. Maybe a good compromise is to put the trailing `|` on its own line instead?

File: _posts/2020-03-31-ion-hash-js-2_0-released.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: news_item
+title: "Ion Hash JS 2.0 Released"
+date: 2020-03-31
+categories: news
```
Comment (zslayton) [inline _posts/2020-03-31-ion-hash-js-2_0-released.md]: Does this need an `ion-hash-js` category tag?

File: libs.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ description: "The latest news about Amazon Ion and the Amazon Ion community."
 |------|----------------|------|---------------|
 | ion-c | [1.0.2](https://github.com/amzn/ion-c/releases/latest) (November 19, 2018) | [Link](https://github.com/amzn/ion-c) | [Link](https://amzn.github.io/ion-c/) |
 | ion-java | [1.5.0](https://github.com/amzn/ion-java/releases/tag/com_amazon_ion_v1.5.0) (June 26, 2019) | [Link](https://github.com/amzn/ion-java) | [Link](https://www.javadoc.io/doc/software.amazon.ion/ion-java/) |
```
Comment (zslayton) [inline libs.md]: While you're in here, could you bump the `ion-java` version number to `1.6.0`?

**Review by zslayton — APPROVED** [review]
LGTM. Some small considerations below.

File: _posts/2020-02-26-ion-java-1_6_0-released.md (hunk 1 above)
Comment (pbcornell) [reply _posts/2020-02-26-ion-java-1_6_0-released.md:12]: Turns out this does change the rendered view.  Note how the "buttons" are oriented [here](http://amzn.github.io/ion-docs/news/ion-java/2020/02/26/ion-java-1_6_0-released.html) vs. [here](http://amzn.github.io/ion-docs/news/2019/05/23/ion-hash-released.html).  Leaving as-is.

File: _posts/2020-03-31-ion-hash-js-2_0-released.md (hunk 2 above)
Comment (pbcornell) [reply _posts/2020-03-31-ion-hash-js-2_0-released.md]: Afaik, we're not really using categories.  I've added "ion-hash-js" to follow the pattern, no harm in doing so.

File: libs.md (hunk 3 above)
Comment (pbcornell) [reply libs.md]: Checked others as well, bumped to the latest version where the page was out-of-date.

**Pull request merged**